docs/schema.md

```diff
@@ -0,0 +1,186 @@
+# Schema Registry
+
+KoP has implemented the Schema Registry to support Kafka clients with [Confluent's Schema serializers and deserializers](https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#supported-formats).
+
+To enable the schema registry, you should add the following configuration:
+
+```properties
+kopSchemaRegistryEnable=true
+```
+
+The Schema Registry listens on the 8001 port by default, which means you should add the following property to create your Kafka producer or consumer.
+
+```properties
+schema.registry.url=http://<broker-ip>:8001
+```
+
+> **Note**
+>
+> See [here](./configuration.md) for other configurations, like how to change the port it listens on.
+
+To manage the schemas, see the [Confluent's Schema REST API](https://docs.confluent.io/platform/current/schema-registry/develop/api.html#compatibility).
+
+## Example: Use KoP Schema Registry with Confluent's Avro serializer
```

Comment:
It seems that your example contains the following steps:

1. Enable the Schema Registry service on KoP
2. Define the Avro schema
3. Basic end-to-end example with Confluent's Avro serializer
4. Query the created schema

Among these 4 steps, the 3rd one is not a task title (I mean using a verb + noun format). It would be better to update the title for step 3. Therefore, these 4 sections keep the same format.